src/use-format-block.hook.js

```diff
@@ -6,10 +6,44 @@ export function useFormatBlock() {
   return {
     formatBlock(value) {
       if (value === document.queryCommandValue('formatBlock')) {
-        performCommandWithValue('div')
+        if (!!window.chrome && isInHTMLList()) {
+          removeHeader()
+        }
+        else {
+          performCommandWithValue('div')
+        }
       } else {
         performCommandWithValue(value)
       }
     },
   }
+}
+
+function isInHTMLList() {
+  let containerNode = window.getSelection().getRangeAt(0).commonAncestorContainer
+  let index = 0
+  let isList = false
+  while(isList == false && index < 2) {
+    if (containerNode.nodeType === Node.ELEMENT_NODE && containerNode.tagName === 'LI') {
+      return true
+    }
+    containerNode = containerNode.parentNode
+    index++
+  }
+}
+
+function removeHeader() {
+  let nodeToBeRemoved = window.getSelection().getRangeAt(0).commonAncestorContainer
+  let index = 0
+  while (nodeToBeRemoved.tagName !== 'H1' && index < 5) {
+    nodeToBeRemoved = nodeToBeRemoved.parentNode
+    index ++
+  }
+  if (index === 4) {
+    throw new Error(`The element does not appear to be wrapped by a header`)
+  }
+  while (nodeToBeRemoved.firstChild) {
+    nodeToBeRemoved.parentNode.insertBefore(nodeToBeRemoved.firstChild, nodeToBeRemoved);
+  }
+  nodeToBeRemoved.parentNode.removeChild(nodeToBeRemoved);
```

Comment:
We can just do `nodeToBeRemoved.remove()`, since bandicoot doesn't support IE11 and `.remove()` is supported by all other browsers. See https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove.

However, calling `.remove()` or `.removeChild()` is an operation that does not go onto the "undo/redo stack" for the browser. Which means that Cmd + Z would not be able to undo what they just did. In the rest of bandicoot, we've stuck to maintaining the undo/redo stack in all places that we can. The way to make this "undoable" is to perform your work with `execCommand` instead of manually modifying the dom.

I think that the better way to remove the node would be to do the following:
```js
window.getSelection().removeAllRanges()
const range = new Range()
range.selectNode(nodeToBeRemoved)
window.getSelection().addRange(range)
document.execCommand('delete')
```

Reply:
I agree, would be nice to preserve c/p. Can you take a look because I am unable to get this to work: https://codepen.io/devcon/pen/aezgWN?editors=1111

I need to remove only one node and replace its children. It should work with `document.execCommand', null, newHTML)` but I am unable to get the `ol` or `h1` into the selection. So I thought to use a combination: exec 'delete' then exec 'insert'. That almost works but I have to use exec 'delete' multiple times which then gives me weird things. Not sure how to proceed.

Reply:
Does this do what you want? https://codepen.io/joeldenning/pen/XvNXRx?editors=1111.

Reply:
Negative, still creates a mess in Chrome 
![image](https://user-images.githubusercontent.com/9112811/62726092-a8a2f280-b9d3-11e9-8ac7-a0f849549133.png)

Reply:
hmm.. not sure best approach here then

Reply:
My thinking is to update [header docs ](https://bandicoot.js.org/walkthrough/headers.html) describing the dilemma and recommend using font size. We have our hands tied by the browsers and their implementation. Will include [stackoberflow](https://stackoverflow.com/questions/57468855/how-can-i-toggle-contenteditable-h1-inside-of-ordered-lists-in-chrome)

Reply:
That sounds good. 👍

Close this PR then in the meantime?